docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/cgov_blog.module

```diff
@@ -218,3 +219,32 @@ function cgov_blog_preprocess_block(&$variables) {
   }
 
 }
+
+/**
+ * Implements hook_preprocess_node().
+ */
+function cgov_blog_preprocess_node(&$variables) {
+  // Add blog series cache tag for blog page.
+  if ($variables['node']->getType() === 'cgov_blog_post' && $variables['view_mode'] == 'full') {
+    $blog_series_value = $variables['node']->get('field_blog_series')->getValue();
+    if ($blog_series_value != NULL && $blog_series_value[0]['target_id'] > 0) {
+      $variables['#cache']['tags'][] = 'node:' . $blog_series_value[0]['target_id'];
+    }
+  }
+}
+
+/**
+ * Implements hook_taxonomy_term_update().
+ */
+function cgov_blog_taxonomy_term_update($term_entity) {
+  /* When taxonomy term for blog categories is updated invalidate the
+   *  corresponding blog series page.
+   */
+  if ($term_entity->getVocabularyId() === "cgov_blog_topics") {
+    $owner_blog = $term_entity->get('field_owner_blog')->getValue();
+    // Invalidate the cache tag for blog series node.
+    if ($owner_blog != NULL && $owner_blog[0]['target_id'] > 0) {
```

Comment:
If you know that $owner_blog should have only 1 owner, you can actually do:
`$owner_blog->target_id` 
The magic __get method gets the first item in a list.